clang/lib/Driver/ToolChains/Repo.cpp

```diff
@@ -0,0 +1,198 @@
+    CmdArgs.push_back(
+        Args.MakeArgString(CRTPath + "/clang_rt.crtbegin-x86_64.o"));
+    CmdArgs.push_back(
+        Args.MakeArgString(CRTPath + "/clang_rt.crtend-x86_64.o"));
+  }
+
+  //----------------------------------------------------------------------------
+  // Library Search Paths
+  //----------------------------------------------------------------------------
+  CmdArgs.push_back(Args.MakeArgString(StringRef("-L") + MuslRoot + "/lib"));
+
+  if (RTC.getVFS().exists(CRTPath))
+    CmdArgs.push_back(Args.MakeArgString("-L" + CRTPath));
+
+  for (const auto &LibPath : RTC.getFilePaths())
+    CmdArgs.push_back(Args.MakeArgString(StringRef("-L") + LibPath));
+
+  //----------------------------------------------------------------------------
+  //
+  //----------------------------------------------------------------------------
+  Args.AddAllArgs(CmdArgs,
+                  {options::OPT_T_Group, options::OPT_e, options::OPT_s,
+                   options::OPT_t, options::OPT_u_Group});
+
+  //----------------------------------------------------------------------------
+  // Libraries
+  //----------------------------------------------------------------------------
+  if (!Args.hasArg(options::OPT_nostdlib, options::OPT_nodefaultlibs)) {
+    CmdArgs.push_back("-lclang_rt.builtins-x86_64");
+  }
+
+  if (D.CCCIsCXX() && RTC.ShouldLinkCXXStdlib(Args)) {
+    RTC.AddCXXStdlibLibArgs(Args, CmdArgs);
+  }
+}
+
+void repo::Linker::ConstructJob(Compilation &C, const JobAction &JA,
+                                const InputInfo &Output,
+                                const InputInfoList &Inputs,
+                                const ArgList &Args,
+                                const char *LinkingOutput) const {
+
+  const ToolChain &TC = getToolChain();
+  ArgStringList CmdArgs;
+
+  AddLinkerInputs(TC, Inputs, Args, CmdArgs, JA);
+  CmdArgs.push_back("-o");
+  CmdArgs.push_back(Output.getFilename());
+
+  if (getToolChain().getTriple().isMusl()) {
+    constructRepoMuslLinkArgs(
+        C, JA,
+        static_cast<const toolchains::RepoMuslToolChain &>(getToolChain()),
+        Output, Inputs, Args, CmdArgs, LinkingOutput);
+  }
+  C.addCommand(std::make_unique<Command>(
+      JA, *this, ResponseFileSupport::AtFileCurCP(),
+      Args.MakeArgString(TC.GetLinkerPath()), CmdArgs, Inputs));
+}
+
+RepoMuslToolChain::RepoMuslToolChain(const Driver &D,
+                                     const llvm::Triple &Triple,
+                                     const llvm::opt::ArgList &Args)
+    : ToolChain(D, Triple, Args) {
+  getProgramPaths().push_back(D.getInstalledDir());
+  if (D.getInstalledDir() != D.Dir)
+    getProgramPaths().push_back(D.Dir);
+
+  if (!D.SysRoot.empty()) {
+    SmallString<128> P(D.SysRoot);
+    llvm::sys::path::append(P, "lib");
+    getFilePaths().push_back(std::string(P.str()));
+  }
+
+  getFilePaths().push_back(D.Dir + "/../lib");
+}
+
+RepoMuslToolChain::~RepoMuslToolChain() {}
```

Comment:
Change `{}` to `= default;`.

Reply:
Fixed it in the commit [7c20e553](https://github.com/SNSystems/llvm-project-prepo/pull/158/commits/7c20e55309a449948d30e3fbcc7ffe4ed3361198).